package.json

```diff
@@ -19,15 +19,16 @@
     "express": "^4.17.1",
     "express-async-handler": "^1.1.4",
     "helmet": "^3.21.1",
-    "mongoose": "^5.7.5"
+    "mongoose": "^5.7.5",
+    "opensubtitles-api": "https://github.com/js-kyle/opensubtitles-api/#RemoteFiles"
```

Comment:
We should change this to npm or Github once bug fixes are merged to master on the main project